### DISCUSSION TITLE
[FEAT] Disable color palette

### DISCUSSION THREAD

**Discussion by OverStyleFR** [fastfetch-cli/fastfetch#1188] [Q&A] [answered]
## Wanted features:

I want an option to disable this :
![image](https://github.com/user-attachments/assets/57f63ffa-92df-48c7-af5a-5be5e19b1668)

## Motivation:

When i execute 'fastfetch --logo none' there still the color palette on the bottom.

**Comment by CarterLi** [comment, marked as answer]
You may remove `Colors` module in your config file

**Reply by sharunkumar** [reply]
> You may remove `Colors` module in your config file

is there a way to disable this using a command line flag?

**Comment by OverStyleFR** [comment]
> You may remove `Colors` module in your config file

I don't want to remove all the colors, just the group of colors at the bottom

**Comment by CarterLi** [comment]
`Colors` is the module name of the group of colors at the bottom

**Comment by OverStyleFR** [comment]
Ha okay, thanks ^^
